test/DynamoCoreWpfTests/ViewExtensions/GraphNodeManagerViewExtensionTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
not sure how this keeps happening...

Reply:
I think @avidit added an editor config - perhaps it has some line endings preference.